XML/TestCases/FunctionalTests-KernelDebug.xml

```diff
@@ -12,4 +12,17 @@
             <SetupType>OneVM</SetupType>
         </SetupConfig>
     </test>
+    <test>
+        <testName>VERIFY-HPC-VM</testName>
+        <testScript>VERIFY-HPC-VM.ps1</testScript>
```

Comment:
Update this to Linux script name

Reply:
Corresponding to the above comment, I will keep this form for future development.